jest.config.js

```diff
@@ -5,4 +5,10 @@ module.exports = {
   coveragePathIgnorePatterns: ['/helpers/', '/node_modules/'],
   collectCoverage: true,
   setupFilesAfterEnv: ['jest-matcher-specific-error'],
+  globals: {
+    'ts-jest': {
+      'diagnostics': false,
```

Comment:
These configs disable some type-checking during tests to make them faster and not requiring them to be error-free to run.
Note that builds will catch compilation errors when building.